content/influxdb/cloud-dedicated/reference/sql/join.md

```diff
@@ -0,0 +1,265 @@
+---
+title: JOIN clause
+description: > 
+    Use the `JOIN` clause to join to data from different tables together.
+menu:
+  influxdb_cloud_dedicated:
+    name: JOIN clause
+    parent: SQL reference
+weight: 202
+---
+
+Use the `JOIN` clause to join to data from different tables together based on
+logical relationships.
+
+- [Syntax](#syntax)
+- [Join types](#join-types)
+  - [INNER JOIN](#inner-join)
+  - [LEFT [OUTER] JOIN](#left-outer-join)
+  - [RIGHT [OUTER] JOIN](#right-outer-join)
+  - [FULL [OUTER] JOIN](#full-outer-join)
+- [Troubleshoot joins](#troubleshoot-joins)
+
+## Syntax
+
+```sql
+SELECT_clause
+FROM <left_join_items>
+[INNER | LEFT [OUTER] | RIGHT [OUTER] | FULL [OUTER]] JOIN <right_join_items>
+ON <join_condition>
+[WHERE_clause]
+[GROUP_BY_clause]
+[HAVING_clause]
+[ORDER_BY_clause]
+```
+
+### Arguments
+
+- **left_join_items**: One or more tables specified in the `FROM` clause that
+  represent the left side of the join.
+- **right_join_items**: One or more tables specified in the `JOIN` clause that
+  represent the right side of the join.
+- **join_condition**: A predicate expression in the `ON` clause that uses the
+  `=` (equal to) comparison operator to compare column values from the left side
+  of the join to column values on the right side of the join. Rows with values
+  that match the defined predicate are joined using the specified
+  [join type](#join-types).
+
+<div id="specify-table"></div>
+
+{{% note %}}
+If both sides of the join include columns with the same name, you need to
+include the table name with the column name to prevent ambiguity.
+Use dot notation to reference the table and column--for example:
```

Comment:
Should we link to ["fully-qualified names"](https://docs.influxdata.com/influxdb/cloud-dedicated/reference/influxql/select/#measurement_expression) for more info, especially for Serverless (with retention policies)?

Reply:
Ah, yeah, good call.

Reply:
I didn't include the link to fully-qualified measurement names since those are unique to InfluxQL. This is just for SQL and table DBRP-mapped table names are actually different than in InfluxQL. For example, in SQL, a mapped table would be `"db_name/rp_name"."column_name"`.